Engine/EditorUI/Panel/InspectorSubpanel/PanelComponent.cpp

```diff
@@ -1035,6 +1085,12 @@ void PanelComponent::ShowAddNewComponentButton()
 			App->editor->selected_game_object->CreateComponent(Component::ComponentType::PARTICLE_SYSTEM);
 		}
 
+		sprintf_s(tmp_string, "%s Trail", ICON_FA_SHARE);
+		if (ImGui::Selectable(tmp_string))
+		{
+			App->editor->selected_game_object->CreateComponent(Component::ComponentType::TRAIL);
+		}
+
```

Comment:
This is related to what I said in another comment, atm you can only create `ComponentTrail` but not `ComponentTrailRenderer`. So, leave just the  `ComponentTrailRenderer` and transform `ComponentTrail` to `Trail`